Samples/auth/Outlook-Add-in-SSO-NAA-Identity/README.md

```diff
@@ -0,0 +1,249 @@
+
+```javascript
+const express = require('express');
+const cors = require('cors');
+
+const app = express();
+
+app.use(cors());
+```
+
+### Access token validation
+
+On the web API side, the **validateJwt** function in `/API/server-helpers/validation-helper.js` verifies the incoming access token's signature and validates it's payload against the `issuer` and `audience` claims.
+
+For validation and debugging purposes, developers can decode **JWT**s (*JSON Web Tokens*) using [jwt.ms](https://jwt.ms).
+
+### Access to data
+
+Controllers should check if the presented access token has the necessary permissions to access the data, depending on the type of permission. This is illustrated in [todolist.js](./API/controllers/todolist.js):
+
+```JavaScript
+exports.getTodos = (req, res, next) => {
+    // Check that caller has the delegated todolist.read permission from the user.
+    if (hasRequiredDelegatedPermissions(req.authInfo, authConfig.protectedRoutes.todolist.delegatedPermissions.read)) {
+
+        try {
+            const owner = req.authInfo['oid'];
+
+            const todos = db.get('todos')
+                .filter({ owner: owner })
+                .value();
+
+            res.status(200).send(todos);
+        } catch (error) {
+            next(error);
+        }
+    } else {
+        next(new Error('User does not have the required permissions.'));
+    }
+}
+```
+
+When granting access to data based on scopes, be sure to follow [the principle of least privilege](https://docs.microsoft.com/azure/active-directory/develop/secure-least-privileged-access).
+
+## More information
+
+Learn more about the Microsoft identity platform:
+
+- [Microsoft identity platform (Microsoft Entra ID for developers)](https://docs.microsoft.com/azure/active-directory/develop/)
+- [Overview of Microsoft Authentication Library (MSAL)](https://docs.microsoft.com/azure/active-directory/develop/msal-overview)
+- [Understanding Microsoft Entra application consent experiences](https://docs.microsoft.com/azure/active-directory/develop/application-consent-experience)
+- [Understand user and admin consent](https://docs.microsoft.com/azure/active-directory/develop/howto-convert-app-to-be-multi-tenant#understand-user-and-admin-consent)
+- [Microsoft identity platform and OpenID Connect protocol](https://docs.microsoft.com/azure/active-directory/develop/v2-protocols-oidc)
+- [Microsoft identity platform ID Tokens](https://docs.microsoft.com/azure/active-directory/develop/id-tokens)
+
+For more information about how OAuth 2.0 protocols work in this scenario and other scenarios, see [Authentication Scenarios for Microsoft Entra ID](https://docs.microsoft.com/azure/active-directory/develop/authentication-flows-app-scenarios).
```

Comment:
```suggestion
For more information about how OAuth 2.0 protocols work in this scenario and other scenarios, see [Microsoft identity platform app types and authentication flows](/entra/identity-platform/authentication-flows-app-scenarios).
```

Reply:
In Github these links have to be the full URL.

Reply:
Ack so sorry about that! Don't know what I was thinking. I will resolve those suggestions.